### ISSUE TITLE
Instagram stopped working

### ISSUE THREAD

**Issue by skoften** [spicywebau/craft-embedded-assets#151]
![Screenshot 2020-11-02 at 21 03 11](https://user-images.githubusercontent.com/1286410/97913639-e31e4b00-1d4e-11eb-86ac-65d0ced86f5b.jpg)

![Screenshot 2020-11-02 at 21 03 01](https://user-images.githubusercontent.com/1286410/97913640-e3b6e180-1d4e-11eb-8be8-79aea55de168.jpg)

Maybe related to https://github.com/spicywebau/craft-embedded-assets/issues/150 ?

**Comment by DavidKabelitz** [comment]
The same here, still get the need to Login Informations even with tokens in use.

**Comment by sjcallender** [comment]
I wonder if it's this: https://github.com/spicywebau/craft-embedded-assets/issues/150#issuecomment-733025585

**Comment by DavidKabelitz** [comment]
Nope not working in our case because we need the texts from Social Media not the embedded link. All texts show the Login hint.

**Comment by skoften** [comment]
What happens when you manually do a curl get on your server, with the access_token, like this:

```
curl -X GET "https://graph.facebook.com/v8.0/instagram_oembed?url=https://www.instagram.com/p/fA9uwTtkSN/&access_token=xxxxxxxxxxx366bcadb8472eced"
```
Can you check the output? Should return some data, including the embed code if successful.

**Comment by DavidKabelitz** [comment]
Yes with a new access token i will get data back. Token will be invalid after 30 days.

**Comment by ttempleton** [comment]
Just released Embedded Assets 2.4.5, which increased the Embed v3 minimum version requirement to 3.4.13, from the previous 3.4.9; [Embed 3.4.11](https://github.com/oscarotero/Embed/releases/tag/v3.4.11) fixed some issues with Instagram/Facebook, so it's worth retrying with the newer version.

If it still doesn't work after updating, though, I would recommend following the instructions from https://github.com/spicywebau/craft-embedded-assets/issues/150#issuecomment-723916346.
